backend/controllers/social.controller.ts

```diff
@@ -0,0 +1,51 @@
+import { Request, Response } from 'express';
+import HttpError from '../utils/error';
+import StatusCode from '../utils/StatusCode'
+import Social, { ISocial } from '../models/social.model';
+
+export const getAll = async (_req: Request, res: Response) => {
+  try {
+    const socials = await Social.find();
+    if (socials.length === 0) {
+      throw new HttpError('Not found any socials', StatusCode.NO_CONTENT);
+    }
+    res.json(socials);
+  } catch (error) {
+    console.log(error);
+    res.status(error.code || StatusCode.INTERNAL_SERVER_ERROR).json({ message: error.message });
+  }
+}
+
+export const getOne = async (req: Request, res: Response) => {
+  const socialId = req.params.id;
+  try {
+    const social = await Social.findOne({ _id: socialId });
```

Comment:
When we can't find record with specific ID, we are getting status code 200 (OK) and `null` as a response. I expect to see an error with proper status code (maybe 404 ?)